tests/Integration/certs/ca/b236481c.0

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Again, can't you use what's in `Core/Security/test/certs` for this files and all the others ?

Reply:
At first it did, but I got a remark that I have to transfer it to tests/Integration/..

Reply:
Ah ?! The tests maybe, but not the test files I would say. They belong to unit tests. Who told you that ?

Reply:
Me :) Because the test was invoked as an integration test. Can it run as a unit test in Travis?

Reply:
The tests I do not think so. But I am just saying that the fake certificates and everything that he is using, instead of creating new one, he could just use the ones I created (and documented how to maintain ;-) ) for unit tests. And these unit tests run on travis.

Reply:
so, keep this files? or remove and use in Core/Security/test/certs?

Reply:
@TaykYoku  can you use the same fake certificates that @chaen mentions?

Reply:
Done